src/main/java/com/hivemq/cli/commands/shell/ShellCommand.java

```diff
@@ -70,12 +70,15 @@
 
     private static boolean exitShell = false;
 
-    private final DefaultCLIProperties defaultCLIProperties;
+    private DefaultCLIProperties defaultCLIProperties;
 
     @SuppressWarnings("NullableProblems")
     @CommandLine.Spec
     private @NotNull CommandLine.Model.CommandSpec spec;
 
+    public ShellCommand() {
```

Comment:
explanation: PicoCLIs ReflectionConfigGenerator will throw an exception if there is no default constructor. not sure if more changes are necessary here.